src/Components/Catalog/Film/Film.js

```diff
@@ -1,17 +1,18 @@
 import { Link } from 'react-router-dom';
+
 import style from '../../../styles/Film.module.css'
 
+const Film = ({film,setModalActive,setFilmID}) => {
 
-const Film = ({film}) => {
     return ( 
         <li className={style.poster}>
             <img className={style.image} src = {film.poster} alt = "Poster"/> 
             <p className={style.title}>{film.title}</p>
             <p className={style.year}>{film.year}</p>
             <Link to= {`/FilmPage/${film.id}`} className={style.info}>Info</Link>
-            <Link to='/remove' className={style.remove}>Remove</Link>
+            <button className={style.remove} onClick={() => {setModalActive(true); setFilmID(film.id)}}>Remove</button>
```

Comment:
create separate function for onClick handler and just pass like `<button onCLick={clickHandler} ....>.....</button>`